WordPress/Classes/Models/Publicizer.m

```diff
@@ -10,4 +10,20 @@ @implementation Publicizer
 @dynamic order;
 @dynamic blog;
 
+- (BOOL)isConnected
+{
+    return self.connectionID > 0;
+}
+
+- (NSInteger)connectionID
+{
+    for (NSDictionary *connection in self.blog.connections) {
+        if ([connection[@"service"] isEqualToString:self.service]) {
+            NSInteger connectionID = [connection[@"conn_ID"] integerValue];
```

Comment:
Please, check out `NSDictionary+SafeExpectations`. We use that fellow quite a lot all over. 'stringForKey:' and `numberForKey:` may help!

(Absolutely nothing wrong with this snippet, but just for peace of mind!)